docs/guides/image-optimization/imageurl-image-optimization.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Use Image Optimization with imageUrl
+description: Learn how to migrate an application using Auth.js to use Clerk for authentication.
+---
+
+# Use Image Optimization with `imageUrl`
+
+Some types in Clerk's JavaScript SDK, such as [`User`](/docs/references/javascript/user/user), [`PublicUserData`](/docs/references/javascript/types/public-user-data#properties), and [`Organization`](/docs/references/javascript/organization/organization#properties), have an `imageUrl`. The URL returned from this property can be scaled down with the following Image Optimization options:
+
+- `"width"`: Sets the max width of the image in pixels.
+- `"height"`: Sets the max height of the image in pixels.
+- `"fit"`: Describes how the image should fit its container. It can take the following values:
+    - `"scale-down"`: The image will scale down to fit the sizes specified in `"width"` and `"height"` if it's bigger, but will not scale up if it's smaller.
+    - `"crop"`: The image will scale down and be cropped to fit within the area specified in `"width"` and `"height"`.
+- `"quality"`: Specifies the image quality for JPEG, WebP, PNG, and AVIF files. Accepts values from `1` to `100`. Defaults to `85`.
+
+## Example
+
+The following example demonstrates how you can get the `imageUrl` from currently active user in a session, and display their profile picture using Clerk's image optimization options:
+
+<CodeBlockTabs type="framework" options={["Next.js App", "Next.js Pages", "React"]}>
```

Comment:
App Router example needs a "use client" , no ?
And we can drop the Pages Router example. 

```suggestion
<CodeBlockTabs type="framework" options={["Next.js", "React"]}>
```

Reply:
Runs for me without "use client"